lib/rails_admin/support/csv_converter.rb

```diff
@@ -32,6 +32,7 @@ def initialize(objects = [], schema = {})
     end
 
     def to_csv(options = {})
+      options = HashWithIndifferentAccess.new(options)
```

Comment:
Calling [`symbolize_keys!`](http://api.rubyonrails.org/classes/Hash.html#method-i-symbolize_keys-21) would be a bit simpler, I guess.